pkg/storage/storage.go

```diff
@@ -159,6 +158,11 @@ func GetPlatformStorage(listers *regopclient.Listers) (imageregistryv1.ImageRegi
 			Claim: defaults.PVCImageRegistryName,
 		}
 		replicas = 1
+	case configapiv1.OvirtPlatformType:
+		cfg.PVC = &imageregistryv1.ImageRegistryConfigStoragePVC{
+			Claim: defaults.PVCImageRegistryName,
+		}
+		replicas = 1
```

Comment:
Does oVirt storage support RWX?

Reply:
No - that would be a shared disk from RHV side. Do we have other options?

Reply:
I thought replica 1 would work with RWO?

Reply:
We want to have at least 2 replicas when it's possible. But in this case the operator can have only 1 replica, so it's ok.

Reply:
Noted. We would track this and use this case as a main driver for supporting RWX